spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SparkTableUtil.java

```diff
@@ -497,9 +525,11 @@ private static void importUnpartitionedSparkTable(SparkSession spark, TableIdent
    * @param spec a partition spec
    * @param stagingDir a staging directory to store temporary manifest files
    * @param checkDuplicateFiles if true, throw exception if import results in a duplicate data file
+   * @param listPartitionParallelism the parallelism to list the datafile of source spark table
```

Comment:
Perhaps we should label this, "Max number of concurrent files to read per partition while indexing table"?